wp-content/plugins/modeltheme-framework/inc/demo-importer-v2/extensions/wbc_importer/demo-data/Metaverse/theme-options.txt
{"last_tab":"13","enefti-enable-breadcrumbs":"1","breadcrumbs-delimitator":"\/","mt_preloader_status":"1","mt_preloader_bg_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"dynamic_sidebars":[""],"enefti_global_link_styling":{"regular":"#d01498","hover":"#d01498","active":"#484848"},"enefti_style_main_texts_color":"#D01498","enefti_style_main_backgrounds_color":{"from":"#647ECB","to":"#D01498"},"enefti_style_main_backgrounds_color_hover":"#FFFFFF","enefti_style_semi_opacity_backgrounds":{"color":"#f02222","alpha":".95","rgba":"rgba(240,34,34,0.95)"},"enefti_text_selection_color":"#FFFFFF","enefti_text_selection_background_color":"#D01498","enefti_nav_menu_color":"#FFFFFF","enefti_nav_menu_color_hover":"#FFFFFF","enefti_nav_submenu_background":"#FFFFFF","enefti_nav_submenu_color":"#484848","enefti_nav_submenu_hover_background_color":"#FFFFFF","enefti_nav_submenu_hover_text_color":"#D01498","enefti_google_fonts_select":["Montserrat:regular,500,600,700,800,900,latin","Poppins:300,regular,500,600,700,latin-ext,latin,devanagari","Raleway:100,200,300,regular,500,600,700,800,900,latin"],"enefti-body-typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_heading_h1":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"36px","line-height":"36px","letter-spacing":"","color":"#242424"},"enefti_heading_h2":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"30px","line-height":"30px","letter-spacing":"","color":"#242424"},"enefti_heading_h3":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"24px","line-height":"24px","letter-spacing":"","color":"#242424"},"enefti_heading_h4":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"18px","line-height":"18px","letter-spacing":"","color":"#242424"},"enefti_heading_h5":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"14px","line-height":"14px","letter-spacing":"","color":"#242424"},"enefti_heading_h6":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"12px","line-height":"12px","letter-spacing":"","color":"#242424"},"enefti_inputs_typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_buttons_typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_heading_h1_tablets":{"font-size":"28px","line-height":"32px"},"enefti_heading_h2_tablets":{"font-size":"26px","line-height":"30px"},"enefti_heading_h3_tablets":{"font-size":"24px","line-height":"28px"},"enefti_heading_h4_tablets":{"font-size":"22px","line-height":"26px"},"enefti_heading_h5_tablets":{"font-size":"20px","line-height":"23px"},"enefti_heading_h6_tablets":{"font-size":"18px","line-height":"21px"},"enefti_heading_h1_smartphones":{"font-size":"26px","line-height":"30px"},"enefti_heading_h2_smartphones":{"font-size":"24px","line-height":"28px"},"enefti_heading_h3_smartphones":{"font-size":"22px","line-height":"26px"},"enefti_heading_h4_smartphones":{"font-size":"20px","line-height":"24px"},"enefti_heading_h5_smartphones":{"font-size":"18px","line-height":"22px"},"enefti_heading_h6_smartphones":{"font-size":"16px","line-height":"20px"},"enefti_css_editor":"#header{margin: 0 auto;}\t\t\t\t\r\n@media only screen and (max-width: 767px) {\r\n.account-urls {\r\n    margin-bottom: 30px;\r\n    margin-top: 10px !important;\r\n}}\r\n@media only screen and (max-width: 767px) {\r\nimg.portfolio_post_image {\r\n    height: 68px !important;\r\n    width: 21% !important;\r\n}}\t\r\n\r\n\r\n.wcfmmp_become_vendor_link {\r\n    margin-top: 10px;\r\n}","enefti_fields_styling_radius":"5","header_layout":"second_header","nav_main_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_top_header2_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_bottom_header2_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_top_header3_color":{"background-color":"#1C1F26","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_bottom_header3_color":"#FFFFFF","inquiry_button_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text":"Post Project","inquiry_button_link":"#","inquiry_button_background_8":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text_8":"Inquiry","inquiry_button_link_8":"#","inquiry_button_background_9":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text_9":"Inquiry","inquiry_button_link_9":"#","enefti_top_bar_text":"Discover, find and sell extraordinary NFT with us.","inquiry_button_text_2":"Create NFT","inquiry_button_link_2":"https:\/\/enefti.modeltheme.com\/create-digital-asset\/","is_header_transparent":"1","is_nav_sticky":"","is_popup_enabled":"","is_search_enabled":"1","search_for":"products","enefti_logo":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/svg\/logo.svg","id":"","height":"","width":"","thumbnail":""},"logo_max_width":"150","enefti_favicon":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/favicon-enefti.png","id":"","height":"","width":"","thumbnail":""},"enefti_top_header_info_switcher":"","discout_header_background":{"background-color":"#f5f5f5","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"discout_header_text":"New Student Deal..","discout_header_date":"22\/02\/2022","discout_header_btn_text":"Join Now","discout_header_btn_link":"#","discout_header_btn_color":"#ef6f31","enefti_mobile_burger_select":"dropdown","enefti_header_category_menu_mobile":"","enefti_header_mobile_switcher_top":"1","enefti_header_mobile_switcher_top_search":"1","enefti_header_mobile_switcher_top_account":"1","enefti_header_mobile_switcher_footer":"","enefti_header_mobile_switcher_footer_search":"1","enefti_header_mobile_switcher_footer_account":"1","enefti-enable-footer-top":"1","footer_top_background":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_top_color_text":"#FFFFFF","enefti-enable-footer-widgets":"1","enefti_number_of_footer_columns":"4","footer_row_1_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti-enable-footer-widgets-row2":"0","enefti_number_of_footer_columns_row2":"4","footer_row_2_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti-enable-footer-widgets-row3":"","enefti_number_of_footer_columns_row3":"4","footer_row_3_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti_footer_text_left":"Copyright by ModelTheme. All Rights Reserved.","enefti_footer_text_right":"Elite Author on ThemeForest.","enefti_card_icons1":{"background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_bottom_background":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_bottom_color_text":"#FFFFFF","footer_bottom_color_links":"#CCCCCC","enefti_backtotop_status":"1","enefti_backtotop_bg_color":{"background-color":"#d01498","background-repeat":"no-repeat","background-size":"","background-attachment":"","background-position":"center center","background-image":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/mt-to-top-arrow.svg","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_contact_phone":"","enefti_contact_email":"","enefti_work_program":"","enefti_contact_address":"","enefti_blog_layout":"enefti_blog_right_sidebar","enefti_blog_layout_sidebar":"sidebar-1","enefti_single_blog_layout":"enefti_blog_right_sidebar","enefti_single_blog_sidebar":"sidebar-1","enefti-blog-post-typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":"","font-weight":"400","font-style":"","font-size":"17px","line-height":"25px","color":"#565656"},"post_featured_image":"1","enefti_shop_layout":"enefti_shop_left_sidebar","enefti_shop_grid_list_switcher":"grid","enefti_shop_layout_sidebar":"woocommerce","enefti-shop-columns":"3","enefti-archive-secondary-image-on-hover":"","enefti_archive_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_layout_version":"main","enefti-enable-general-info":"","enefti-enable-general-img1":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-general-desc1":"Lorem ipsum dolor sit amet, consectetur adipisc elit. Duis sollicitudin diam in diamui varius, sed anim.","enefti-enable-general-img2":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-general-desc2":"Lorem ipsum dolor sit amet, consectetur adipisc elit. Duis sollicitudin diam in diamui varius, sed anim.","enefti_project_color":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_project_color_text":"#FFFFFF","enefti_project_image":"","enefti_view_counter":"","enefti_single_product_layout":"enefti_shop_fullwidth","enefti_single_shop_sidebar":"sidebar-1","enefti-enable-related-products":"1","enefti-related-products-number":"4","img_404":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/uploads\/2022\/02\/404.png","id":"6896","height":"1099","width":"1924","thumbnail":"https:\/\/enefti.modeltheme.com\/wp-content\/uploads\/2022\/02\/404-150x150.png"},"enefti-enable-popup":"","enefti-enable-popup-img":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-popup-company":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/logo-enefti.png","id":"","height":"","width":"","thumbnail":""},"enefti-enable-popup-desc":"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sit amet sagittis sem, at sollicitudin lectus.","enefti-enable-popup-form":"","enefti-enable-popup-additional":"","enefti-enable-popup-expire-date":"1","enefti-enable-popup-show-time":"5000","enefti_social_fb":"http:\/\/facebook.com","enefti_social_tw":"google","enefti_social_pinterest":"","enefti_social_skype":"","enefti_social_instagram":"http:\/\/instagram.com","enefti_social_youtube":"","enefti_social_dribbble":"","enefti_social_linkedin":"http:\/\/linkedin.com","enefti_social_deviantart":"http:\/\/deviantart.com","enefti_social_digg":"","enefti_social_flickr":"","enefti_social_stumbleupon":"","enefti_social_tumblr":"","enefti_social_vimeo":"","enefti_social_share_links":{"twitter":"1","facebook":"1","whatsapp":"1","pinterest":"","linkedin":"","telegram":"","email":""},"enefti_social_share_locations":{"product":"1","post":"1"},"redux-backup":1}

wp-content/plugins/modeltheme-framework/inc/demo-importer-v2/extensions/wbc_importer/demo-data/NFT Musician/theme-options.txt
{"last_tab":"1","enefti-enable-breadcrumbs":"1","breadcrumbs-delimitator":"\/","mt_preloader_status":"1","mt_preloader_bg_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_preloader_color":"#FFFFFF","dynamic_sidebars":[""],"enefti_global_link_styling":{"regular":"#d01498","hover":"#d01498","active":"#484848"},"enefti_style_main_texts_color":"#D01498","enefti_style_main_backgrounds_color":{"from":"#647ECB","to":"#D01498"},"enefti_style_main_backgrounds_color_hover":"#FFFFFF","enefti_style_semi_opacity_backgrounds":{"color":"#f02222","alpha":".95","rgba":"rgba(240,34,34,0.95)"},"enefti_text_selection_color":"#FFFFFF","enefti_text_selection_background_color":"#D01498","enefti_nav_menu_color":"#FFFFFF","enefti_nav_menu_color_hover":"#FFFFFF","enefti_nav_submenu_background":"#FFFFFF","enefti_nav_submenu_color":"#484848","enefti_nav_submenu_hover_background_color":"#FFFFFF","enefti_nav_submenu_hover_text_color":"#D01498","enefti_google_fonts_select":["Montserrat:regular,500,600,700,800,900,latin","Poppins:300,regular,500,600,700,latin-ext,latin,devanagari","Raleway:100,200,300,regular,500,600,700,800,900,latin"],"enefti-body-typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_heading_h1":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"36px","line-height":"36px","letter-spacing":"","color":"#242424"},"enefti_heading_h2":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"30px","line-height":"30px","letter-spacing":"","color":"#242424"},"enefti_heading_h3":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"24px","line-height":"24px","letter-spacing":"","color":"#242424"},"enefti_heading_h4":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"18px","line-height":"18px","letter-spacing":"","color":"#242424"},"enefti_heading_h5":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"14px","line-height":"14px","letter-spacing":"","color":"#242424"},"enefti_heading_h6":{"font-family":"Poppins","font-options":"","google":"1","font-backup":"","font-size":"12px","line-height":"12px","letter-spacing":"","color":"#242424"},"enefti_inputs_typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_buttons_typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":""},"enefti_heading_h1_tablets":{"font-size":"28px","line-height":"32px"},"enefti_heading_h2_tablets":{"font-size":"26px","line-height":"30px"},"enefti_heading_h3_tablets":{"font-size":"24px","line-height":"28px"},"enefti_heading_h4_tablets":{"font-size":"22px","line-height":"26px"},"enefti_heading_h5_tablets":{"font-size":"20px","line-height":"23px"},"enefti_heading_h6_tablets":{"font-size":"18px","line-height":"21px"},"enefti_heading_h1_smartphones":{"font-size":"26px","line-height":"30px"},"enefti_heading_h2_smartphones":{"font-size":"24px","line-height":"28px"},"enefti_heading_h3_smartphones":{"font-size":"22px","line-height":"26px"},"enefti_heading_h4_smartphones":{"font-size":"20px","line-height":"24px"},"enefti_heading_h5_smartphones":{"font-size":"18px","line-height":"22px"},"enefti_heading_h6_smartphones":{"font-size":"16px","line-height":"20px"},"enefti_css_editor":"#header{margin: 0 auto;}\t\t\t\t\r\n@media only screen and (max-width: 767px) {\r\n.account-urls {\r\n    margin-bottom: 30px;\r\n    margin-top: 10px !important;\r\n}}\r\n@media only screen and (max-width: 767px) {\r\nimg.portfolio_post_image {\r\n    height: 68px !important;\r\n    width: 21% !important;\r\n}}\t\r\n\r\n\r\n.wcfmmp_become_vendor_link {\r\n    margin-top: 10px;\r\n}","enefti_fields_styling_radius":"5","header_layout":"second_header","nav_main_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_top_header2_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_bottom_header2_color":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_top_header3_color":{"background-color":"#1C1F26","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"mt_style_bottom_header3_color":"#FFFFFF","inquiry_button_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text":"Post Project","inquiry_button_link":"#","inquiry_button_background_8":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text_8":"Inquiry","inquiry_button_link_8":"#","inquiry_button_background_9":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"inquiry_button_text_9":"Inquiry","inquiry_button_link_9":"#","enefti_top_bar_text":"Discover, find and sell extraordinary NFT with us.","inquiry_button_text_2":"Create NFT","inquiry_button_link_2":"https:\/\/enefti.modeltheme.com\/create-digital-asset\/","is_header_transparent":"1","is_nav_sticky":"","is_popup_enabled":"","is_search_enabled":"1","search_for":"products","enefti_logo":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/svg\/logo.svg","id":"","height":"","width":"","thumbnail":""},"logo_max_width":"150","enefti_favicon":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/favicon-enefti.png","id":"","height":"","width":"","thumbnail":""},"enefti_top_header_info_switcher":"","discout_header_background":{"background-color":"#f5f5f5","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"discout_header_text":"New Student Deal..","discout_header_date":"22\/02\/2022","discout_header_btn_text":"Join Now","discout_header_btn_link":"#","discout_header_btn_color":"#ef6f31","enefti_mobile_burger_select":"dropdown","enefti_header_category_menu_mobile":"","enefti_header_mobile_switcher_top":"1","enefti_header_mobile_switcher_top_search":"1","enefti_header_mobile_switcher_top_account":"1","enefti_header_mobile_switcher_footer":"","enefti_header_mobile_switcher_footer_search":"1","enefti_header_mobile_switcher_footer_account":"1","enefti-enable-footer-top":"1","footer_top_background":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_top_color_text":"#FFFFFF","enefti-enable-footer-widgets":"1","enefti_number_of_footer_columns":"4","footer_row_1_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti-enable-footer-widgets-row2":"0","enefti_number_of_footer_columns_row2":"4","footer_row_2_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti-enable-footer-widgets-row3":"","enefti_number_of_footer_columns_row3":"4","footer_row_3_spacing":{"units":"px","padding-top":"0","padding-right":"","padding-bottom":"0","padding-left":""},"enefti_footer_text_left":"Copyright by ModelTheme. All Rights Reserved.","enefti_footer_text_right":"Elite Author on ThemeForest.","enefti_card_icons1":{"background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_bottom_background":{"background-color":"#000000","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"footer_bottom_color_text":"#FFFFFF","footer_bottom_color_links":"#CCCCCC","enefti_backtotop_status":"1","enefti_backtotop_bg_color":{"background-color":"#d01498","background-repeat":"no-repeat","background-size":"","background-attachment":"","background-position":"center center","background-image":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/mt-to-top-arrow.svg","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_contact_phone":"","enefti_contact_email":"","enefti_work_program":"","enefti_contact_address":"","enefti_blog_layout":"enefti_blog_right_sidebar","enefti_blog_layout_sidebar":"sidebar-1","enefti_single_blog_layout":"enefti_blog_right_sidebar","enefti_single_blog_sidebar":"sidebar-1","enefti-blog-post-typography":{"font-family":"Raleway","font-options":"","google":"1","font-backup":"","font-weight":"400","font-style":"","font-size":"17px","line-height":"25px","color":"#565656"},"post_featured_image":"1","enefti_shop_layout":"enefti_shop_left_sidebar","enefti_shop_grid_list_switcher":"grid","enefti_shop_layout_sidebar":"woocommerce","enefti-shop-columns":"3","enefti-archive-secondary-image-on-hover":"","enefti_archive_background":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_layout_version":"main","enefti-enable-general-info":"","enefti-enable-general-img1":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-general-desc1":"Lorem ipsum dolor sit amet, consectetur adipisc elit. Duis sollicitudin diam in diamui varius, sed anim.","enefti-enable-general-img2":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-general-desc2":"Lorem ipsum dolor sit amet, consectetur adipisc elit. Duis sollicitudin diam in diamui varius, sed anim.","enefti_project_color":{"background-color":"","background-repeat":"","background-size":"","background-attachment":"","background-position":"","background-image":"","media":{"id":"","height":"","width":"","thumbnail":""}},"enefti_project_color_text":"#FFFFFF","enefti_project_image":"","enefti_view_counter":"","enefti_single_product_layout":"enefti_shop_fullwidth","enefti_single_shop_sidebar":"sidebar-1","enefti-enable-related-products":"1","enefti-related-products-number":"4","img_404":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/uploads\/2022\/02\/404.png","id":"6896","height":"1099","width":"1924","thumbnail":"https:\/\/enefti.modeltheme.com\/wp-content\/uploads\/2022\/02\/404-150x150.png"},"enefti-enable-popup":"","enefti-enable-popup-img":{"url":"","id":"","height":"","width":"","thumbnail":""},"enefti-enable-popup-company":{"url":"https:\/\/enefti.modeltheme.com\/wp-content\/themes\/enefti\/images\/logo-enefti.png","id":"","height":"","width":"","thumbnail":""},"enefti-enable-popup-desc":"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sit amet sagittis sem, at sollicitudin lectus.","enefti-enable-popup-form":"","enefti-enable-popup-additional":"","enefti-enable-popup-expire-date":"1","enefti-enable-popup-show-time":"5000","enefti_social_fb":"http:\/\/facebook.com","enefti_social_tw":"google","enefti_social_pinterest":"","enefti_social_skype":"","enefti_social_instagram":"http:\/\/instagram.com","enefti_social_youtube":"","enefti_social_dribbble":"","enefti_social_linkedin":"http:\/\/linkedin.com","enefti_social_deviantart":"http:\/\/deviantart.com","enefti_social_digg":"","enefti_social_flickr":"","enefti_social_stumbleupon":"","enefti_social_tumblr":"","enefti_social_vimeo":"","enefti_social_share_links":{"twitter":"1","facebook":"1","whatsapp":"1","pinterest":"","linkedin":"","telegram":"","email":""},"enefti_social_share_locations":{"product":"1","post":"1"},"redux-backup":1}

wp-content/themes/nftdaddy/config.php
<?php
$dev_content_w = 900;

/** Script version - Modify it if you got cached css & js */
$dev_script_version = '1.0';

/** JS scripts - Notice: Do not add jquery.js, WP automatically handle it before these files  */
$dev_js_scripts = array(
    array(
        'name' => 'be-js-main',
        'src' => '/assets/js/jquery.min.js'
    ),
    array(
        'name' => 'be-js-easing',
        'src' => '/assets/js/jquery.easing.min.js'
    ),
    array(
        'name' => 'be-js-slick',
        'src' => '/assets/js/slick.min.js'
    ),
    array(
        'name' => 'be-js-magnific',
        'src' => '/assets/js/jquery.magnific-popup.min.js'
    ),
    array(
        'name' => 'be-js-aos',
        'src' => '/assets/js/aos.js'
    ),
    array(
        'name' => 'be-js-clipboard',
        'src' => '/assets/js/clipboard.min.js'
    ),
    array(
        'name' => 'be-js-countdown',
        'src' => 'assets/js/jquery.countdown.min.js'
    ),
    array(
        'name' => 'be-js-waypoints',
        'src' => 'assets/js/jquery.waypoints.min.js'
    ),
    array(
        'name' => 'be-js-scroll',
        'src' => 'assets/js/infinite-scroll.pkgd.js'
    ),
    array(
        'name' => 'be-js-counterup',
        'src' => 'assets/js/jquery.counterup.min.js'
    ),
    array(
        'name' => 'be-js-dataTables',
        'src' => 'assets/js/jquery.dataTables.min.js'
    ),
    array(
        'name' => 'be-js-app',
        'src' => 'assets/js/app.js'
    ),
    array(
        'name' => 'project-js-app',
        'src' => 'project/customizer.js'
    ),
);

/** include styles */
$dev_styles = array(
    array(
        'name' => 'magnific',
        'src' => 'assets/css/magnific-popup.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'slick',
        'src' => 'assets/css/slick.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'font-aos',
        'src' => 'assets/css/aos.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'dev-dataTables',
        'src' => 'assets/css/jquery.dataTables.min.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'dev-tailwind',
        'src' => 'assets/css/tailwind.min.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'dev-custom',
        'src' => 'assets/css/custom.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'dev-responsive',
        'src' => 'assets/css/responsive.css',
        'screen' => 'all'
    ),
    array(
        'name' => 'project-dev-custom',
        'src' => 'project/dev_custom.css',
        'screen' => 'all'
    ),
);
/* BACK-END START EDITING HERE */

/** post thumbnail sizes (px) - this is the thumbnail image of blog posts */
$dev_post_thumbnail_size = array(
    'width' => 400,
    'height' => 300,
    'crop' => true
);

/** Additional image sizes (px) */
$dev_add_sizes = array(
    // 'home_block_items' => array('width' => 600, 'height' => 600, 'crop' => true),
    'customer_care' => array('width' => 300, 'height' => 300, 'crop' => true),
    'banner_page' => array('width' => 1170, 'height' => 500, 'crop' => true),
    'image_overview' => array('width' => 273, 'height' => 273, 'crop' => true),

);

/** Copyright name & url */
$dev_copyright_name = 'NFTDADDY';
$dev_copyright_url = 'https://nftdaddy.net';

/** Theme support */

$dev_post_formats = array(
    'aside',
);
global $dev_textdomain;
$dev_textdomain = 'nftdaddy';

/** Navigation menu */
$dev_nav_menus = array(
    'main' => __('Main Menu', $dev_textdomain),
    'footer' => __('Footer Menu', $dev_textdomain)
);


/* OK, that's enough, Stop editing */
